docs/workload-development-kit/fabric-data-plane.md

```diff
@@ -12,41 +12,98 @@ ms.date: 05/21/2024
+[Microsoft Fabric OneLake](../onelake/index.yml) is a unified, logical data lake for the entire organization, designed to be the single place for all analytics data. It comes automatically with every Microsoft Fabric tenant and is built on top of Azure Data Lake Storage (ADLS) Gen2. OneLake supports any type of file, structured or unstructured, and stores all tabular data in Delta Parquet format. It allows for collaboration across different business groups by providing a single data lake that is governed by default with distributed ownership for collaboration within a tenant's boundaries. Workspaces within a tenant enable different parts of the organization to distribute ownership and access policies, and all data in OneLake can be accessed through data items such as [Lakehouses and Warehouses](../data-warehouse/data-warehousing.md).
 
 In terms of data stores, OneLake serves as the common storage location for ingestion, transformation, real-time insights, and Business Intelligence visualizations. It centralizes the different Fabric services and is the storage for data items consumed by all workloads in Fabric.
 
 ## How to read and write data in Microsoft Fabric 
 
 Microsoft Fabric is a platform that enables the storage and management of customer data. In order to read and write data in Fabric, you need to use the [Fabric REST APIs](/rest/api/fabric/articles/) and the appropriate authentication methods.
 
+### API Permissions
+
+Some methods of accessing customer data require the use of other services outside of Fabric such as Azure Storage or Azure SQL Database. For example, in the Microsoft Fabric Developer kit sample, the API permission Azure Storage user_impersonation is used in conjunction with the PowerBI Service Lakehouse.Read.All permission in order to access data from Lakehouses.
+
+You may choose to use Azure SQL Database in order to access table data from Warehouse items. In this case you would configure your app with Azure SQL Database user_impersonation in order to query the database on behalf of the user and PowerBI Service Warehouse.Read.All in order to query the Fabric REST API Get Warehouse endpoint.
+
+Make sure you configure your Microsoft Entra ID app according to your development needs.
```

Comment:
What does this sentence mean?

Reply:
I think that the intent here is to emphasize that if you need a certain service, such as Azure SQL Database, you likely need to make changes to your app and configure it accordingly. So we want them to pay attention to what delegated scopes are required for the actions they want.

Reply:
Ok. It's not very clear that this is your intent. Perhaps something like this:

Add the necessary scopes to your Microsoft Entra ID app so that you'll have the right permissions for the Fabric operations your app requires.